sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/TokenResult.java

```diff
@@ -59,6 +70,45 @@ public String partition() {
         return this.partition;
     }
 
+    /**
+     * Get the token expiration time
+     * @return the token expiration value
+     */
+    public Date expiresOn() {
```

Comment:
getExpiryDate?

Reply:
based on other property is using the get method with `get`... thus won't change here.